_config.php

```diff
@@ -0,0 +1,2 @@
+<?php
+define('BBCODEPARSER_PATH', __DIR__);
```

Comment:
I think the convention is that this would be that this is called `BBCODEPARSER_DIR` - the path would be `BASE_PATH . BBCODEPARSER_DIR`